temporian/core/evaluator.py

```diff
@@ -33,15 +33,23 @@
 def evaluate(
     query: Query,
     input_data: Data,
-    backend: AvailableBackends = "numpy",
+    verbose: int = 0,
+    check_execution: bool = True,
```

Comment:
Default to False / add TODO to change it to False before public release?

Reply:
Sure.

*Default to False + add todo.